invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/menu.overrides

```diff
@@ -250,7 +250,7 @@
       #rdm-close-burger-menu-icon {
         position: absolute;
         top: 2rem;
-        right: 0;
+        right: 2rem;
```

Comment:
if you are fixing also misalignment with the searchbar, it would be easier to set common padding on the parent element